src/entities/MovieForm/MovieForm.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from "react";
+import "./MovieForm.css";
+import { genresList } from "../../entities/SelectTabs/selectTabsMock";
+
+export const MovieForm = ({ initialMovie, onFormSubmit }) => {
+  const [movie, setMovie] = useState({
+    title: initialMovie?.title ?? "",
+    movieUrl: initialMovie?.movieUrl ?? "",
+    genre: initialMovie?.genre ?? "",
+    overview: initialMovie?.overview ?? "",
+    releaseDate: initialMovie?.releaseDate ?? "",
+    rating: initialMovie?.rating ?? "",
+    runtime: initialMovie?.runtime ?? "",
+    overview: initialMovie?.overview ?? "",
+  });
```

Comment:
Names are the same as in `initialMovie` object, why do we need an extra object? 
```suggestion
  const [movie, setMovie] = useState(initialMovie);
```

Reply:
done

Reply:
I don't see any changes